monitoring/ingestion/ingestion-global-dashboard.json

```diff
@@ -203,6 +224,126 @@
       "transformations": [],
       "type": "stat"
     },
+    {
+      "datasource": "${DS_PROMETHEUS}",
+      "description": "How many instances are currently online and offline",
+      "fieldConfig": {
+        "defaults": {
+          "displayName": "",
+          "mappings": [],
+          "noValue": "0",
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              }
+            ]
+          },
+          "unit": "/${stateful_replicas}"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 4,
+        "w": 6,
+        "x": 12,
+        "y": 0
+      },
+      "id": 27,
+      "options": {
+        "colorMode": "value",
+        "graphMode": "none",
+        "justifyMode": "center",
+        "orientation": "vertical",
+        "reduceOptions": {
+          "calcs": [
+            "last"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "text": {},
+        "textMode": "auto"
+      },
+      "pluginVersion": "7.5.7",
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "sum(quorum_size{pod=\"$pod\",job=\"${job_zookeeper}\",namespace=\"${namespace}\"})",
```

Comment:
How does `$pod` get assigned?

Reply:
Pod is auto generated by the operator I believe

Reply:
Removed pod var as discussed